aws/resource_aws_elasticache_replication_group_test.go

```diff
@@ -1035,10 +1075,10 @@ resource "aws_elasticache_replication_group" "bar" {
     parameter_group_name = "default.redis3.2.cluster.on"
     automatic_failover_enabled = true
     cluster_mode {
-      replicas_per_node_group = 1
-      num_node_groups = 2
+      num_node_groups         = %[2]d
+      replicas_per_node_group = %[3]d
```

Comment:
There’s a bug in govet where it won’t raise an error for these if you’re using the numbered index, but will if you’re using them as the actual type eg “%d” (when specifying the wrong type). Up to you, but I’d suggest there’s value in listing these variables explicitly (or by assigning them to a Terraform variable above and referencing that) so that these errors are raised?

Reply:
I wish I could fix that bug. 😉  For now I'll revert back to non-indexed references.